README.md

```diff
@@ -90,6 +91,78 @@ Sure! However you will need a [SPID account](https://www.agid.gov.it/en/platform
 
 If you have some time to spare and wish to get involved on a regular basis, [contact us](mailto:federico.feroldi@pagopa.it).
 
+### What permissions are used by the IO app?
+
+Because different platform have different types of Permissions below we have two sections about permissions requested by the IO app for both environment (iOS and Android). Some permissions may be defined but not used. Their presence is due to dependencies with third-party modules or because they are required by the target store.
```

Comment:
platform -> platforms
environment -> environments

Yes, it's me <sub><sup>nitpicking grammar on Github</sup></sub> (⌐■_■)

Reply:
```suggestion
Because different platforms have different types of Permissions below we have two sections about permissions requested by the IO app for both environments (iOS and Android). Some permissions may be defined but not used. Their presence is due to dependencies with third-party modules or because they are required by the target store.
```